.github/workflows/build-dev-app.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       (github.event.action == 'labeled' && github.event.label.name == 'dev-app preview') ||
       (github.event.action == 'synchronize' && contains(github.event.pull_request.labels.*.name, 'dev-app preview'))
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@629c2de402a417ea7690ca6ce3f33229e27606a5 # renovate: tag=v2.0.0
```

Comment:
Is this the correct SHA? I can't find it upstream, should it be `ec3a7ce113134d7a93b817d10a8272cb61118579`?

Reply:
Thanks, I don't know how I messed up those. Probably a copy-paste bug  _facepalm_.